inference-engine/tests/ie_test_utils/common_test_utils/data_utils.hpp

```diff
@@ -155,6 +155,39 @@ fill_data_random(T *pointer, std::size_t size, const uint32_t range = 10, int32_
     }
 }
 
+template <typename T>
+void inline fill_random_unique_sequence(T* rawBlobDataPtr,
```

Comment:
Not sure that I can understand purpose  of the routine from its name. Why we need it? How its behavior differs from fill_data_roi? Can we add a comment with detailed description?

Reply:
it's an templated overload of original `fill_random_unique_sequence()` which changes first argument from type InferenceEngine::Blob::Ptr to raw pointer T*, the functionality is fill the data area with unique random data (without any repeats in value and also in ascending order. I followed the `fill_data_random()` pattern to add this overload because bucketize requires second inputs to be unique ascending random data.

Reply:
comment added